pipeline/api/cloud.py

```diff
@@ -531,7 +531,8 @@ def run_pipeline(
             compute_type=compute_type,
             compute_requirements=compute_requirements,
         )
-        return self._post("/v2/runs", json.loads(run_create_schema.json()))
+        run_json: dict = self._post("/v2/runs", json.loads(run_create_schema.json()))
+        return RunGet.parse_obj(run_json)
```

Comment:
This would be a breaking change in the API (i.e. clients would need to update their code when upgrading to this version). Therefore, if we want to go ahead with this we would need to ensure we make this a major update (e.g bump version to 1.0.0)

Reply:
Just did a VB to 0.1.0

Reply:
I think _strictly_ this should be a bump to 1.0.0 (see https://semver.org/)

Reply:
Okay!